# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1513877738 18000
#      Thu Dec 21 12:35:38 2017 -0500
# Node ID fe54e9633815a24b8f7b75e5603f31c968b83ec2
# Parent  53ca76c5cc8d6f2ae39bfbead9ecd100c4cabe39
lex.ll: Fix macro for implicit fallthrough warning.

diff --git a/libinterp/parse-tree/lex.ll b/libinterp/parse-tree/lex.ll
--- a/libinterp/parse-tree/lex.ll
+++ b/libinterp/parse-tree/lex.ll
@@ -42,17 +42,17 @@ object) relevant global values before an
    // Disable these warnings for code that is generated by flex,
    // including pattern rules.  Push the current state so we can
    // restore the warning state prior to functions we define at
    // the bottom of the file.
 #  pragma GCC diagnostic push
 #  pragma GCC diagnostic ignored "-Wold-style-cast"
 #  pragma GCC diagnostic ignored "-Wsign-compare"
 #  pragma GCC diagnostic ignored "-Wzero-as-null-pointer-constant"
-#  if defined (HAVE_WARNING_IMPLICIT_FALLTHROUGH)
+#  if defined (HAVE_WARN_IMPLICIT_FALLTHROUGH)
 #    pragma GCC diagnostic ignored "-Wimplicit-fallthrough"
 #  endif
 #endif
 
 // Define away the deprecated register storage class specifier to avoid
 // potential warnings about it.
 #if ! defined (register)
 #  define register
